<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MSTSYS0000443" failures="1" tests="1" name="Default test" time="142.692" errors="0" timestamp="2022-03-28T14:05:34 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="139.431">
    <failure type="java.lang.AssertionError" message="Index 0 out of bounds for length 0">
      <![CDATA[java.lang.AssertionError: Index 0 out of bounds for length 0
at org.testng.Assert.fail(Assert.java:97)
at stepDefinition.StepDefinitionForEmployeeSide.user_clicks_the_link_in_the_email_and_redirected_to_the_portal_page(StepDefinitionForEmployeeSide.java:74)
at âœ½.User clicks the link in the email and redirected to the portal page(file:///C:/Users/MSTSYS/eclipse-workspace/AdvanceLevelAutomation/Featurefile/TestCase.feature:24)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
